core/isolate.rs

```diff
@@ -331,7 +336,11 @@ impl Isolate {
     let op = if let Some(ref f) = isolate.dispatch {
       f(op_id, control_buf.as_ref(), PinnedBuf::new(zero_copy_buf))
```

Comment:
Since op_id == 0 has special meaning, we should assert in the case of the dispatch that we are not using it. Please add
```rust
assert!(op_id != 0);
```